assets/src/components/activities/common/Evaluation.tsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
```

Comment:
I pulled out each of the delivery components to be reused across activities and made the `shouldShow` conditions clearer since they caused a few bugs in the past.